pkgs/stdenv/linux/bootstrap-tools/scripts/unpack-bootstrap-tools.sh

```diff
@@ -17,6 +17,15 @@ else
    LD_BINARY=$out/lib/ld-*so.?
 fi
 
+# path to version-specific libraries, like libstdc++.so
+LIBSTDCXX_SO_DIR=$(echo $out/lib/gcc/*/*)
```

Comment:
I checked that for all 16 of our existing `bootstrap-files` tarballs there is only one subdirectory under `lib/gcc`, and only one subdirectory beneath that (`unpack-bootstrap-tools.sh` is a huge pain because every time we touch it we have to rebuild the whole universe).